src/modifiers.rs

```diff
@@ -113,7 +113,7 @@ pub struct Redirect(pub Url);
 impl Modifier<Response> for Redirect {
     fn modify(self, res: &mut Response) {
         let Redirect(url) = self;
-        res.headers.set(headers::Location(format!("{:?}", url)));
+        res.headers.set(headers::Location(format!("{}", url)));
```

Comment:
@ProtectedMode, it must be there, because this impl in http://doc.rust-lang.org/nightly/std/string/trait.ToString.html

```rust
impl<T> ToString for T where T: String, T: ?Sized
```